sdk/eventhub/event-hubs/package.json

```diff
@@ -50,8 +45,8 @@
     "integration-test:browser": "karma start --single-run",
     "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
-    "lint:fix": "eslint package.json tsconfig.json src test samples --ext .ts --fix",
-    "lint": "eslint package.json tsconfig.json src test samples --ext .ts -f html -o event-hubs-lintReport.html || exit 0",
+    "lint:fix": "eslint package.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test --ext .ts -f html -o search-lintReport.html || exit 0",
```

Comment:
Why the change from `tsconfig.json` to `api-extractor.json`?

Reply:
(you had this in the Service Bus one as well, but I forgot to drop a comment in there)

Reply:
Just copying and pasting from the guide at the original issue. We can have both!

Reply:
cc @willmtemple

Reply:
This is the only thing blocking me from approving right now as well.